src/es6/Entity.js

```diff
@@ -48,10 +48,25 @@ export class Entity {
 	/**
 	 * Protected: Add a listener to a subentity of this entity.
 	 * @param {*} entityType A entity class that extends this class.
-	 * @param {*} href Href of the entity to be created
+	 * @param {*} href Href or Entity of the entity to be created
 	 * @param {*} onChange callback function that accepts an {entityType} to be called when subentity changes.
 	 */
 	_subEntity(entityType, href, onChange) {
```

Comment:
New change that gives us the ability to create a sub-entity by a complete entity.

Reply:
sorry was commenting at docs, and the source code is here.